capstone/src/main/java/com/google/sps/data/ApkFileFeatures.java

```diff
@@ -2,23 +2,27 @@
 
 import com.google.appengine.api.datastore.Entity;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class ApkFileFeatures {
```

Comment:
I think this class is powerful but I'm not seeing you use it below, it looks like you're still using a hash map. Is that a followup CL?

Reply:
I implemented a hashmap for mapping the various file types in each apk and storing them in Datastore. However, the ApkFileFeatures helps create a class for each apk and its features retrieved from Datastore so that the features can be easily accessed for visualization from the client side when converted to json.